### PR TITLE
Initial Community Work

### PR DESCRIPTION
**Pull request by kris-nova**
- Adding DOSi paper
 - Adding initial charter
 - Adding Apache2 license (and language)
 - Adding community README

Signed-off-by: Kris Nóva <kris@nivenly.com>

### CODE REVIEW

File: doctrine/02-04-2021-DistributedOperatingSystemVoid.md
Comment (invidian) [inline doctrine/02-04-2021-DistributedOperatingSystemVoid.md:27]: I wonder if being Kubernetes-specific here and below is desired. We could have a statement, that it also applies to other distributed systems and Kubernetes is used more as an real-life example?

File: CHARTER.md
Comment (kris-nova) [reply CHARTER.md:16]: i am a fan of them for `cat` purposes - i am also an avid CLI editior (i did this entire PR from the CLI) so yeah - tell me what we want and i can fix 😄

Comment (kris-nova) [reply CHARTER.md:16]: tldr; i ultimately have weak opinions here and am happy to fix 😄

Comment (invidian) [reply CHARTER.md:16]: Fair enough, no strong opinions here if it's deliberate.

File: CHARTER.md
Comment (kris-nova) [inline CHARTER.md:7]: keep this kubernetes ambiguous

File: doctrine/02-04-2021-DistributedOperatingSystemVoid.md
Comment (kris-nova) [reply doctrine/02-04-2021-DistributedOperatingSystemVoid.md:27]: while i agree that not being kubernetes specific is a good philosophy of the project (as we have it captured here https://github.com/cosi-project/community/pull/5/files#r614302069 in the charter)

this particular document is just a 1:1 copy of my blog https://nivenly.com/lib/2021-04-02-operating-system-interface/

and i *think it makes sense to keep the blog as literal as possible - part of the blog was intended to be very kubernetes specific to address the problem space concretely 

perhaps we can add some sort of meta information on this copy to show that its just a copy of the paper? or perhaps we can just reference the original copy and not add it to this PR?

i thought having a first concrete problem space write up here would be beneficial for the project as a hole :)

Comment (invidian) [reply doctrine/02-04-2021-DistributedOperatingSystemVoid.md:27]: >perhaps we can add some sort of meta information on this copy to show that its just a copy of the paper? or perhaps we can just reference the original copy and not add it to this PR?

I think it make sense to have it here as 1:1 copy (as backup, it's easier to find, grep for etc etc). Perhaps a note then, explaining that this applies to distributed system in general and that Kubernetes is used here as a specific example would be sufficient :+1:

File: CHARTER.md
Comment (andrewrynhard) [reply CHARTER.md:7]: > keep this kubernetes ambiguous

+1

File: CHARTER.md
Comment (andrewrynhard) [inline CHARTER.md:15]: super duper nit: `s/number 1/number one`. I think an English teacher got mad at me for this once, but I couldn't tell you why 😄 .

**Review by andrewrynhard — APPROVED** [review]
We don't have anything setup to enforce this just yet, but thinking to begin with we should try to get a few approvals at least before we merge. LGTM otherwise!

File: doctrine/02-04-2021-DistributedOperatingSystemVoid.md
Comment (kris-nova) [inline doctrine/02-04-2021-DistributedOperatingSystemVoid.md:5]: Meta information about the paper

**Pull request merged**
